src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/DefendantResponseApplicantNotificationHandler.java

```diff
@@ -141,7 +141,8 @@ private void sendNotificationToSolicitorSpec(CaseData caseData, String recipient
 
             if (caseData.getDefenceAdmitPartPaymentTimeRouteRequired() == IMMEDIATELY
                 && (RespondentResponseTypeSpec.FULL_ADMISSION.equals(caseData.getRespondent1ClaimResponseTypeForSpec())
-                || RespondentResponseTypeSpec.FULL_ADMISSION.equals(caseData.getRespondent2ClaimResponseTypeForSpec()))
+                || RespondentResponseTypeSpec.FULL_ADMISSION.equals(
```

Comment:
I cant spot any code change on this line, then why this line change?

Reply:
I think its a formatting change. you can ignore